pkg/virt-launcher/virtwrap/converter/compute/clock.go

```diff
@@ -73,29 +74,29 @@ func convert_v1_Clock_To_api_Clock(source *v1.Clock, clock *api.Clock) error {
 			newTimer := api.Timer{Name: "rtc"}
 			newTimer.Track = string(source.Timer.RTC.Track)
 			newTimer.TickPolicy = string(source.Timer.RTC.TickPolicy)
-			newTimer.Present = boolToYesNo(source.Timer.RTC.Enabled, true)
```

Comment:
Since all the `boolToYesNo` invocations in this file currently have their `defaultYes` parameter set to `true`, how about we create a simplified version?

```go
func boolToYesNoWithYesDefault(value *bool) string {
    if value == nil {
        return "yes"
    }
    
    if *value {
        return "yes"
    }
    return "no"
}
```

Reply:
I don't think we need custom handling when we can use the golang way of implementing the `MarshalXMLAttr` and the `UnmarshalXMLAttr` interfaces from the standard library.